curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md

```diff
@@ -17,10 +17,23 @@ videoId: XzosGWLafrY
 question:
   text: How would you iterate over and print the keys and values of a dictionary named "user"?
   answers:
-    - '<pre>for key in user.items():<br>    print(key)</pre>'
-    - '<pre>for key, value in user.all():<br>    print(key, value)<br>    print(value)'
-    - '<pre>for key, value in user.items():<br>    print(key, value)</pre>'
-    - '<pre>for key, value in user<br>    print(key, value)</pre>'
+    - ```python
```

Comment:
Hey @JapneetSingh5, if you look at the [details of the failed test](https://travis-ci.com/github/freeCodeCamp/freeCodeCamp/jobs/345546388), it says something like this...
````
/home/travis/build/freeCodeCamp/freeCodeCamp/curriculum/challenges/
english/08-data-analysis-with-python/data-analysis-with-python-course/
python-iteration-and-modules.english.md: 16: yaml-linter YAML code blocks 
should be valid [bad indentation of a sequence entry at line 4, column 7:

        - ```python

          ^] [Context: "```yml"]
````

I'm not sure, but I think you might be missing the `|` before the code blocks in the answers?

Reply:
Oh Okay, will check it out! Thanks!